### PR TITLE
Arrows

### PR DESCRIPTION
**Pull request by scott-massey**
Added support for arrows. Currently the positioning doesn't work, so it defaults to center.

### CODE REVIEW

File: src/components/ImageSlider/index.js [hunk 1]
```diff
@@ -50,13 +93,27 @@ class Images extends Component {
...
+  }
+
+  handleLeftArrow = () => {
+    console.log('handleLeftArrow')
+    const { activeIndex } = this.state
+    if (activeIndex === 0) return
+    this.handleChange(activeIndex - 1)
+  }
+
+  handlePress = (i) => () => {
```
Comment (jeremyblalock) [inline src/components/ImageSlider/index.js]: This doesn't match our normal prettier config. Could you just copy the prettier config from another project?

File: src/components/ImageSlider/index.js [hunk 2]
```diff
@@ -50,13 +93,27 @@ class Images extends Component {
...
     let { images } = this.props
 
     images[i] && images[i].action && images[i].action()
   }
 
-  scrollViewRef = el => {
+  scrollViewRef = (el) => {
```
Comment (jeremyblalock) [inline src/components/ImageSlider/index.js]: Same with the above comment. This shouldn't be corrected by prettier like this.

File: src/components/ImageSlider/index.js [hunk 3]
```diff
@@ -83,44 +141,127 @@ class Images extends Component {
     }
 
     let scrollViewStyles = { paddingBottom }
-
+    if (!arrows) return <View></View>
```
Comment (jeremyblalock) [inline src/components/ImageSlider/index.js]: This is not a viable solution. We should have `defaultProps` on `ImageSlider` for `arrows: {}`,

**Comment by zachbharris** [comment]
For the prettier issue, you can add `"arrowParens": "avoid"` to the `.prettierrc`

**Pull request closed without merging**